bucket/outlook-google-calendar-sync.json

```diff
@@ -2,10 +2,9 @@
     "version": "2.8.1-alpha",
     "description": "Sync your Outlook and Google calendars.",
     "license": "MPL-2.0",
-    "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.8.1-alpha/OutlookGoogleCalendarSync-2.8.1-alpha-full.nupkg#/dl.7z",
+    "url": "https://github.com/phw198/OutlookGoogleCalendarSync/releases/download/v2.8.1-alpha/Portable_OGCS_v2.8.1.zip",
```

Comment:
Do not use tags. Here you can see that this file does not exists. Tags does not explicitly mean there is release available for this tag.

![A](https://i.imgur.com/uNbkODU.png)